net/overture/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=overture
+PKG_VERSION:=1.6-rc1
```

Comment:
Is there any reason why you want to add this version, which is in RC?

Reply:
Because I think it's the latest. I will change to latest stable version.